addons/web_tour/static/src/js/tip.js

```diff
@@ -146,7 +225,7 @@ var Tip = Widget.extend({
         return $location;
     },
     _reposition: function () {
-        if (this.tip_opened) return;
+        if (this.tip_opened && !this.moveTip) return;
```

Comment:
@Esperax Just checked what you told me yesterday, seems indeed necessary but way checking the condition at all? I just removed that line and I don't see the problem. Do you have an explanation?

Reply:
removing the entire line will indeed change nothing, but removing the` && !this.moveTip` will allow the bug.
The code that prevent the tip to be moved is the condition that this.tip_opened is true. But when it's open, there is a case (the one I already described) where we want it to move.

Reply:
Precisely why I am asking to remove the line to always reposition when it's opened... would that not work? And allow you to remove `moveTip` from `this`.